Makefile

```diff
@@ -182,6 +182,7 @@ protos: bin/protoc-gen-go-fieldpath
 	find v2 -name '*.pb.go' -exec sh -c 'f={}; mkdir -p $$(dirname "$${f#v2/}"); echo mv $$f $${f#v2/}; mv $$f $${f#v2/}' \;
 	@mv ${TMPDIR}/vendor ${ROOTDIR}
 	@rm -rf ${TMPDIR} v2
+	go-fix-acronym -w -a '^Os' $(shell find api/ core/runtime/ -name '*.pb.go')
```

Comment:
Any particular reason we don't add this to the regex below?

Reply:
No, just for readability

Reply:
Having looked at cramming it all in one, I agree 😄  ❤️